### PR TITLE
[sil_tchad] migration to github repo

### PR DESCRIPTION
**Pull request by andjc**

### CODE REVIEW

File: release/sil/sil_tchad/sil_tchad.keyboard_info [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "license": "mit",
+    "languages": ["amj", "bbz", "shu", "arb", "sjg", "bmi", "bva", "bjv", "bxv", "bes", "bid", "btf", "bvo", "glc", "bvf", "bub", "bdm", "bso", "cds", "dgk", "djc", "dau", "daa", "dai", "dzg", "dsi", "fni", "fgr", "fra", "fub", "fui", "fuv", "fvr", "gab", "gdk", "gid", "gqr", "goy", "glu", "glj", "gvl", "hau", "hed", "hor", "jyy", "jeu", "ksp", "kwg", "kwv", "kvf", "ckq", "kbl", "knc", "kzr", "kth", "klf", "kyq", "ker", "kie", "kia", "kqp", "kou", "vkj", "kxj", "xuo", "kvi", "gdm", "kot", "lap", "lln", "ndy", "mde", "muj", "xmj", "mxf", "mcs", "mge", "mgb", "mpg", "mvu", "mls", "mcn", "msv", "mes", "mdg", "mcw", "mpk", "myb", "zim", "mmy", "mlj", "mou", "bdo", "mpi", "kqx", "mub", "moz", "mvh", "mua", "mse", "mug", "mje", "mne", "nnc", "ndm", "nmc", "sba", "nnn", "nie", "noy", "nzy", "pnz", "lme", "rou", "saa", "sag", "mwm", "swy", "sys", "sok", "sor", "sbj", "tma", "tax", "tuq", "tng", "trj", "tmc", "tug", "tui", "ubi", "zag", "zna", "zrn"],
+    "description": "This keyboard enables the typing of all Chadian languages using the Chadian National Alphabet."
```
Comment (darcywong00) [inline release/sil/sil_tchad/sil_tchad.keyboard_info]: The CI server is expecting these BCP 47 tags
```
[01:55:39][Step 1/1] 'fra' is a valid tag but is not canonical: 'fra' should be 'fr'
[01:55:39][Step 1/1] 'hau' is a valid tag but is not canonical: 'hau' should be 'ha'
[01:55:39][Step 1/1] 'sag' is a valid tag but is not canonical: 'sag' should be 'sg'
```

Comment (andjc) [reply release/sil/sil_tchad/sil_tchad.keyboard_info]: Thanks @darcywong00  when i have time, will write a routine to automatically correct those inside the editor I use

**Comment by LornaSIL** [comment]
It looks to me like you could delete the following files:
* welcome\Clavier Tchad Unicode pour Keyman.pdf
* welcome\Tchad Unicode Keyman Keyboard.pdf
* usage.htm (the welcome.htm is more complete)
* usage-fra.htm (the welcome.htm is more complete)

Changes:
* welcome.htm, welcome-fra.htm, sil_tchad.php files
     * change rule for saltillo (in 2nd row of first table) from ```''``` to ```;'```
     * change rule for reversed glottal (in 3rd table) from ```;?``` to ```:??```
* Mobile layout
     * on the Shift layer for the period, it is outputting ```>```. The easiest way to get around this is just to change the code ```K_PERIOD``` to ```U_002E```
     * On the numeric layer for ```K_BKSLASH``` change to ```U_002F``` (the main key is outputting backslash and it should be forward slash as the keycap shows)
     * I haven't had a chance to test the diacritic layer. The dotted circle (U+25cc) is showing as a square box on my phone which makes everything look horrible. I wonder if we need to ship a font with the keyboard. I'll report more tomorrow.
     * I also want to double check that all the characters in the desktop keyboard are supported in the mobile keyboard and I haven't had a chance to check that.

Nice work!

**Comment by andjc** [comment]
@LornaSIL I will change the OSK font and unset the touch layout font. It doesn't make sense to have TNR as a font for touch screen since it isn't available on most platforms. WE will need to rely on Roboto for Android and Helvetica Neue for iOS8 and San Francisco for iOS9+. Not sure what the character coverage is for these fonts though. But setting touch to TNR will just result in these fonts being used anyway.

**Comment by andjc** [comment]
@LornaSIL Done.
out of curiosity, do we know why they are using the Greek Theta and Greek Chi? Not an issue, just curiosity.

**Comment by LornaSIL** [comment]
@andjc I'm not sure, but I know some of the keyboards try to include the IPA characters needed too.

**Comment by LornaSIL** [comment]
Quote from welcome.htm: (Shaded characters are intended to be used phonetically, not orthographically in Chadian languages.) -- theta and chi are shaded. I'm not sure why the bar h is not shaded though. U+0127.

* Rename ```DEPRECEATED.md``` > ```DEPRECATED.md``` (in \legacy\t\tchadunicode_with_charis_fonts folder)

Everything else looks good.

**Comment by andjc** [comment]
Hi @LornaSIL I left the shading as it was in the original. I can update that if you wish. But not knowing what the author intended, and whether it was intentional or an oversight I left as is

**Pull request merged**